sink/sink.h

```diff
@@ -38,6 +38,9 @@ class Sink : public MiracBroker {
   void Pause();
   void Teardown();
 
+ protected:
+  virtual wfd::Peer* Peer() const override;
```

Comment:
ah I see, but maybe if we encapsulate timer logic within a dedicated class (thus use aggregation instead of inheritance from MiracBroker), we won't need this method.